WebGoat-Legacy-master/src/main/java/org/owasp/webgoat/lessons/BlindScript.java

```diff
@@ -159,7 +159,7 @@ public void executeSpyWare( WebSession s )
         // Lets gather some information about the users browsing history
         String userHome = System.getProperty("user.home" ) + "\\Local Settings\\Temporary Internet Files";
         String separator = System.getProperty("line.separator");
-        File dir = new File(userHome);
+     //   File dir = new File(userHome);
         StringBuffer browserFiles = new StringBuffer();
         if (dir.isDirectory()) {
             File[] children = dir.listFiles();

```

Comment:
**Suggestion:** The commented out line 162 creates a File object 'dir' which is used in line 164. If the line remains commented out, it will cause a compile error as 'dir' is not defined.
```suggestion
     File dir = new File(userHome);
     if (dir.isDirectory()) {
         File[] children = dir.listFiles();
```